crates/solidity/testing/sanctuary/src/tests.rs

```diff
@@ -212,14 +213,13 @@ fn create_bindings(version: &Version, source_id: &str, output: &ParseOutput) ->
         }),
     );
     let parser = Parser::create(version.clone())?;
-    let built_ins_tree = parser
-        .parse(
-            NonterminalKind::SourceUnit,
-            bindings::get_built_ins(version),
-        )
-        .tree();
-    let built_ins_cursor =
-        transform_built_ins_node(&built_ins_tree).cursor_with_offset(TextIndex::ZERO);
+    let built_ins_parser = parser.parse(
```

Comment:
nit: this is a `ParseOutput` object, not `Parser`:

```suggestion
    let built_ins_output = parser.parse(
```